core/src/com/unciv/logic/civilization/diplomacy/CityStateFunctions.kt

```diff
@@ -540,6 +540,17 @@ class CityStateFunctions(val civInfo: Civilization) {
                     diplomacy.setFlag(DiplomacyFlags.BorderConflict, 10)
                 }
             }
+            
+            // If citystate gets tile stolen from another civ, then the civ gets a diplomacy Influence debuffer.
+            if (diplomacy.hasModifier(DiplomaticModifiers.StealingTerritory) && diplomacy.isRelationshipLevelLT(RelationshipLevel.Friend)) {
+                diplomacy.addInfluence(-25f)
```

Comment:
This should be in UniqueTriggerActivation:1107 (UniqueType.OneTimeTakeOverTilesInRadius)
This should happen *when* the tile is being stolen
Also, even if you have a good relationship with a CS, stealing tiles is a Bad Action, and should have consequences
I think 15 influence per tile stolen sounds around right